documentation/content/en/articles/problem-reports/_index.adoc

```diff
@@ -272,7 +272,7 @@ When a problem report has not received attention after several weeks, it is wort
 
 There are a few ways to do so, ideally in the following order, with a few days between attempting each communication channel:
 
-* Find the relevant FreeBSD mailing list for the problem report from the extref:{handbook}[list in the Handbook, eresources-mail] and send a message to that list asking about assistance or comments on the problem report.
+* Send an e-mail to https://docs.freebsd.org/en/books/handbook/eresources/#eresources-mail[the relevant list], seeking comments on the report.
```

Comment:
Hi.

Thanks for your PR.

`extref` is a macro, see [here](https://github.com/freebsd/freebsd-doc/blob/main/shared/lib/CrossDocumentReferencesMacro/extension.rb), we need to document it in the fdp-primer.

You can use it here instead of the full URL.

```
extref:{handbook}eresources/[the relevant list, eresources-mail]
-->
https://docs.freebsd.org/en/books/handbook/eresources/#eresources-mail
```

Reply:
Thanks, I tried that (or something like it) before creating the PR, the preview appears wrong. 

Edition: 

![image](https://user-images.githubusercontent.com/192271/147726432-abed8282-1847-4359-a8e5-58403b35c022.png)

Preview: 

![preview](https://user-images.githubusercontent.com/192271/147726490-34d417c1-adeb-47d9-95d3-c99e803c3416.png)

Please, did I mis-type? Or is the preview misleading?

Reply:
The syntax is correct.

Github preview or tests with asciidoctor standalone won't render it.

To see it properly, you need to build with Hugo. See more [here](https://docs.freebsd.org/en/books/fdp-primer/doc-build/#doc-build-rendering-html).

The `make run` is very helpful when editing documentation.

Reply:
> `make run`

Thanks. That was an ideal hint. Success: 

![image](https://user-images.githubusercontent.com/192271/147817202-757d62ee-8fc6-466f-be45-8b16c3a490a8.png)

----

Rough notes to self: 

* https://gist.github.com/grahamperrin/345d49b94aab7d90239c6ea76995ef28